depthai_demo.py

```diff
@@ -112,46 +130,69 @@ def print_sys_info(info):
     fps = FPSHandler() if conf.useCamera else FPSHandler(cap)
 
     if conf.useCamera or conf.args.sync:
-        pv = PreviewManager(fps, display=conf.args.show, colorMap=conf.getColorMap(), disp_multiplier=disp_multiplier)
+        pv = PreviewManager(fps, display=conf.args.show, colorMap=conf.getColorMap(), disp_multiplier=disp_multiplier, mouseTracker=True)
 
-        if conf.args.camera == "left" or conf.useDepth:
+        if conf.leftCameraEnabled:
             pm.create_left_cam(mono_res, conf.args.mono_fps)
-        if conf.args.camera == "right" or conf.useDepth:
+        if conf.rightCameraEnabled:
             pm.create_right_cam(mono_res, conf.args.mono_fps)
-        if conf.args.camera == "color":
-            pm.create_color_cam(rgb_res, conf.args.rgb_fps, conf.args.full_fov_nn, conf.useHQ)
+        if conf.rgbCameraEnabled:
+            pm.create_color_cam(nn_manager.input_size if conf.useNN else (300, 300), rgb_res, conf.args.rgb_fps, conf.args.full_fov_nn, conf.useHQ)
 
         if conf.useDepth:
             pm.create_depth(
                 conf.args.disparity_confidence_threshold,
                 conf.getMedianFilter(),
+                conf.args.sigma,
                 conf.args.stereo_lr_check,
+                conf.args.lrc_threshold,
                 conf.args.extended_disparity,
                 conf.args.subpixel,
+                useDepth=Previews.depth.name in conf.args.show or Previews.depth_raw.name in conf.args.show,
+                useDisparity=Previews.disparity.name in conf.args.show or Previews.disparity_color.name in conf.args.show,
+                useRectifiedLeft=Previews.rectified_left.name in conf.args.show,
+                useRectifiedRight=Previews.rectified_right.name in conf.args.show,
             )
 
         enc_manager = EncodingManager(pm, dict(conf.args.encode), conf.args.encode_output) if len(conf.args.encode) > 0 else None
 
     if len(conf.args.report) > 0:
         pm.create_system_logger()
 
-    nn_pipeline = nn_manager.create_nn_pipeline(pm.p, pm.nodes, shaves=conf.args.shaves, use_depth=conf.useDepth,
-                                                use_sbb=conf.args.spatial_bounding_box and conf.useDepth,
-                                                minDepth=conf.args.min_depth, maxDepth=conf.args.max_depth,
-                                                sbbScaleFactor=conf.args.sbb_scale_factor)
-    pm.create_nn(nn=nn_pipeline, sync=conf.args.sync)
+    if conf.useNN:
+        nn_pipeline = nn_manager.create_nn_pipeline(pm.p, pm.nodes, shaves=conf.args.shaves, use_depth=conf.useDepth,
```

Comment:
try to be more consistent with naming across the codebase, either camelCase (as core) or snake_case